modal/cli/container.py

```diff
@@ -54,12 +55,15 @@ async def exec(task_id: str, command: str):
         return
 
     client = await _Client.from_env()
-    res: api_pb2.ContainerExecResponse = await client.stub.ContainerExec(
-        api_pb2.ContainerExecRequest(task_id=task_id, command=command, pty_info=get_pty_info(shell=True))
-    )
-    if res.exec_id == "":
-        print(f"Failed to execute command. Is the container ID ({task_id}) correct?")
-        return
+    try:
+        res: api_pb2.ContainerExecResponse = await client.stub.ContainerExec(
+            api_pb2.ContainerExecRequest(task_id=task_id, command=command, pty_info=get_pty_info(shell=True))
+        )
+    except GRPCError as err:
+        if err.status == Status.NOT_FOUND:
+            print(f"Container ID {task_id} not found!")
```

Comment:
Shouldn't this print to stderr?

Also might be better to remove the exclamatory `!`, and return a nonzero exit code

Reply:
I used rich.Console, printed it in red, removed the exclamation mark, and returned a nonzero exit code. (I didn't print to stderr though, only stdout -- is it important to print to stderr?)

Reply:
Yeah, I think stderr is better, since people often use stdout for piping.

You can do `rich.print("...", file=sys.stderr)`